advocacy_docs/edb-postgres-ai/console/estate/reference/metrics_ingestion.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Metrics Ingestion into the EDB Postgres AI Estate
+navTitle: Metrics Ingestion
+description: The metrics that can be ingested into the EDB Postgres AI Estate.
+---
+
+All agents in the EDB Postgres AI Estate can send metrics to the Estate control plane.
```

Comment:
Do we want to talk about the CP?

Reply:
For now, Control Plane is sufficiently technical to refer to the abstract entitity that is the Estate infra.

Suggestion for if we go with Platform/Etate
```suggestion
All agents that you configure in your Estate can send metrics to the EDB Postgres AI Platform.
```